shared/studio/tabs/auth/index.tsx

```diff
@@ -80,6 +90,62 @@ export const authAdminTabSpec: DatabaseTabSpec = {
 
 const secretPlaceholder = "".padStart(32, "•");
 
+const AuthUrls = observer(function AuthUrls({
+  builtinUIEnabled,
+}: {
+  builtinUIEnabled: boolean;
+}) {
+  const instanceState = useInstanceState();
+  const databaseState = useDatabaseState();
+
+  const url = new URL(instanceState.serverUrl);
+  if (url.hostname.endsWith(".edgedb.cloud")) {
+    url.port = "";
+  }
+  url.pathname = `db/${databaseState.name}/ext/auth`;
+
+  const baseUrl = url.toString();
+
+  return (
+    <div className={styles.authUrls}>
+      <div className={styles.label}>OAuth callback endpoint:</div>
+      <CopyUrl url={`${baseUrl}/callback`} />
+      <div
+        className={cn({[styles.disabled]: !builtinUIEnabled})}
+        style={{display: "contents"}}
+      >
+        <div className={styles.label}>Built-in UI sign in url:</div>
+        <CopyUrl url={`${baseUrl}/ui/signin`} />{" "}
+      </div>
```

Comment:
Not a blocker: Might as well add another one of these for `signup` in case people want to link to that directly, too.